osquery/CMakeLists.txt

```diff
@@ -223,8 +223,8 @@ if(NOT DEFINED ENV{SKIP_TESTS})
   # osquery core set of unit tests build with SDK.
   add_executable(osquery_tests main/tests.cpp ${OSQUERY_TESTS})
   TARGET_OSQUERY_LINK_WHOLE(osquery_tests libosquery)
-  target_link_libraries(osquery_tests gtest libosquery_testing)
-  SET_OSQUERY_COMPILE(osquery_tests "${CXX_COMPILE_FLAGS} -DGTEST_HAS_TR1_TUPLE=0")
```

Comment:
This is pretty old: https://github.com/facebook/osquery/pull/702/commits/4f10a35f8007d77b215efcda66c7a5327e792f6f

No context around the addition. If it builds on our supported hosts let's remove it.